src/wp-includes/css/custom-properties.css

```diff
@@ -204,4 +214,78 @@ body {
 	--wp-admin--admin-menu--notification--color: var(--wp-admin--theme--notification--contrast);
 	--wp-admin--admin-menu--notification--background--current: var(--wp-admin--admin-menu--notification--background);
 	--wp-admin--admin-menu--notification--color--current: var(--wp-admin--admin-menu--notification--color);
+
+	/* black grayscale rgba # = 0, 0, 0, # */
+	--wp-admin--themes--black-0: rgba(0, 0, 0, 0);
+	--wp-admin--themes--black-05: rgba(0, 0, 0, 0.05);
+	--wp-admin--themes--black-2: rgba(0, 0, 0, 0.2);
+	--wp-admin--themes--black-5: rgba(0, 0, 0, 0.5);
+	--wp-admin--themes--black-6: rgba(0, 0, 0, 0.6);
+	--wp-admin--themes--black-7: rgba(0, 0, 0, 0.7);
+
+	/* white grayscale rgba */
+	--wp-admin--themes--white-65: rgba(255, 255, 255, 0.65);
+	--wp-admin--themes--white-7: rgba(246, 247, 247, 0.7);
+	--wp-admin--themes--white-9: rgba(240, 240, 241, 0.9);
+
+	/* unique rgba colors */
+	--wp-admin--themes--alpha-dark: rgba(44, 51, 56, 0.7);
+	--wp-admin--themes--alpha-gray: rgba(140, 143, 148, 0.1);
+	--wp-admin--themes--blue: rgba(34, 113, 177, 0.8);
+	--wp-admin--themes--light-blue: rgba(79, 148, 212, 0.8);
```

Comment:
For the most part we've been avoiding using the color name in the properties. Rather than creating these intermediate properties and using them later, could you use them directly below?